packages/react-core/src/components/ChipGroup/ChipGroup.tsx

```diff
@@ -17,112 +19,163 @@ export interface ChipGroupProps extends React.HTMLProps<HTMLDivElement> {
+    onClick: (_e: React.MouseEvent) => undefined as any,
+    closeBtnAriaLabel: 'Close chip group',
+    tooltipPosition: 'top',
+    'aria-label': 'Chip group category'
   };
 
+  componentDidMount() {
+    this.setState({
+      isTooltipVisible: Boolean(
+        this.headingRef.current && this.headingRef.current.offsetWidth < this.headingRef.current.scrollWidth
+      )
+    });
+  }
+
   toggleCollapse = () => {
     this.setState(prevState => ({
-      isOpen: !prevState.isOpen
+      isOpen: !prevState.isOpen,
+      isTooltipVisible: Boolean(
+        this.headingRef.current && this.headingRef.current.offsetWidth < this.headingRef.current.scrollWidth
+      )
     }));
   };
 
-  renderToolbarGroup() {
-    const { isOpen } = this.state;
-    const { headingLevel = 'h4' } = this.props;
-    return (
-      <ChipGroupContext.Provider value={headingLevel}>
-        <InnerChipGroup {...this.props} isOpen={isOpen} onToggleCollapse={this.toggleCollapse} />
-      </ChipGroupContext.Provider>
+  renderLabel(id: string) {
+    const { categoryName, tooltipPosition } = this.props;
+    const { isTooltipVisible } = this.state;
+    return isTooltipVisible ? (
+      <Tooltip position={tooltipPosition} content={categoryName}>
+        <span tabIndex={0} ref={this.headingRef} className={css(styles.chipGroupLabel)} aria-hidden="true" id={id}>
+          {categoryName}
+        </span>
+      </Tooltip>
+    ) : (
+      <span ref={this.headingRef} className={css(styles.chipGroupLabel)} aria-hidden="true" id={id}>
+        {categoryName}
+      </span>
     );
   }
 
-  renderChipGroup() {
-    const { className } = this.props;
+  render() {
+    const {
+      categoryName,
+      children,
+      className,
+      isClosable,
+      closeBtnAriaLabel,
+      'aria-label': ariaLabel,
+      onClick,
+      numChips,
+      expandedText,
+      collapsedText,
+      ...rest
+    } = this.props;
     const { isOpen } = this.state;
-    return (
-      <ul className={css(styles.chipGroup, className)}>
-        <InnerChipGroup {...this.props} isOpen={isOpen} onToggleCollapse={this.toggleCollapse} />
-      </ul>
-    );
-  }
+    const collapsedTextResult = fillTemplate(collapsedText as string, {
+      remaining: React.Children.count(children) - numChips
+    });
 
-  render() {
-    const { withToolbar, children } = this.props;
     if (React.Children.count(children)) {
-      return withToolbar ? this.renderToolbarGroup() : this.renderChipGroup();
+      const renderChipGroup = (id: string) => (
+        <div className={css(styles.chipGroup, className, categoryName && styles.modifiers.category)}>
+          {categoryName && this.renderLabel(id)}
+          <ul
+            className={css(styles.chipGroupList)}
+            {...(categoryName && { 'aria-labelledby': id })}
+            {...(!categoryName && { 'aria-label': ariaLabel })}
+            // aria-labelledby={id}
+            // aria-label={ariaLabel}
+            role="list"
```

Comment:
One last thing -- [after reading MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/List_role) I don't think this is necessary anymore since we're using a `<ul>` which is a list. Per @jessiehuff 's review we can remove it.

Reply:
@jessiehuff @mcoker this was listed as a new attribute to add in the issue.  Should it not be added?

Reply:
I originally thought the same thing since `<ul>` is implicitly `role="list"`, but we found after testing different browser and AT combinations that the explicit role seemed to be beneficial. I'm in favor of keeping it unless we find that it becomes better supported in the future without it. :)